solvers/apg.py

```diff
@@ -14,8 +15,9 @@ class Solver(BaseSolver):
     parameters = {
         'n_inner_iter': [1, 5]
```

Comment:
Why a 'loss' parameter was included to the other solvers but not to this one?

Reply:
I added it here as well